test/sequential/test-cpu-prof.js

```diff
@@ -3,6 +3,13 @@
 // This tests that --cpu-prof and --cpu-prof-path works.
 
 const common = require('../common');
+if (process.features.debug &&
+  process.config.variables.node_code_cache_path === 'yes') {
```

Comment:
ATM this is not the best predicate for embedded code-cache...
I've submitted https://github.com/nodejs/node/pull/27311 to fix the value, and this predicate (it could be any non `''` value if combined with the previous manual embedding technique)